seqr/views/apis/variant_search_api.py

```diff
@@ -240,6 +239,7 @@ def _get_variant_main_transcript_field_val(parsed_variant):
 VARIANT_SAMPLE_DATA = [
     {'header': 'sample', 'value_path': 'sampleId'},
     {'header': 'num_alt_alleles', 'value_path': 'numAlt'},
+    {'header': 'filters', 'process': lambda val: ';'.join(val or []), 'variant_value_path': '[genotypeFilters][*]'},
```

Comment:
what does `variant_value_path` represent here? is this code change for backwards compatibility?

Reply:
yes, for local installs running elasticsearch the UI needs to support both

Reply:
and `value_path ` here represents the path to the field within the genotype, so `variant_value_path` represents the oath to the field within the variant